dataland-framework-toolbox/scripts/convert-excel-to-csv.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+## PARAMETER REFERENCE (https://help.libreoffice.org/latest/en-GB/text/shared/guide/csv_params.html)
+# 44 - Use ',' to seperate fields
+# 34 - Use '"' to format text
+# UTF-8 - Use UTF-8 output formatting
+# 1 - Start at the first line
+# BLANK - No special column formatters
+# 1033 - Use "English - United States" Formatting
+# false - Quoted field as text --> NO (default)
+# true - Detect special numbers (default)
+# false - Save cell contents as shown --> No
+# false - Export cell formulas --> No
+# false - Remove spaces --> No
+# -1 - Export all sheets
+
+libreoffice --headless \
```

Comment:
Issues here for windows users. Had to run "/c/Program Files/LibreOffice/program"/scalc.exe.  Maybe add comments?